From f875e8ccaa8b0d1cb4033de294061a4b21387d3f Mon Sep 17 00:00:00 2001
From: John Greene <jogreene@redhat.com>
Date: Mon, 7 Jul 2014 15:40:52 -0400
Subject: [redhat] RHMAINTAINERS: adjust ixgbe/ixgbevf maintainer

Message-id: <1404747652-12992-1-git-send-email-jogreene@redhat.com>
Patchwork-id: 85286
O-Subject: [RHEL 7 PATCH] [redhat] RHMAINTAINERS: adjust ixgbe/ixgbevf maintainer
Bugzilla:
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Neil Horman <nhorman@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/RHMAINTAINERS b/redhat/RHMAINTAINERS
index 84818d1..302c06e 100644
--- a/redhat/RHMAINTAINERS
+++ b/redhat/RHMAINTAINERS
@@ -281,12 +281,12 @@ S: Maintained
 F: drivers/net/ethernet/intel/igbvf/
 
 INTEL IXGBE ETHERNET DRIVER (ixgbe)
-M: Andy Gospodarek <agospoda@redhat.com>
+M: John Greene <jogreene@redhat.com>
 S: Maintained
 F: drivers/net/ethernet/intel/ixgbe/
 
 INTEL IXGBEVF ETHERNET DRIVER (ixgbevf)
-M: Andy Gospodarek <agospoda@redhat.com>
+M: John Greene <jogreene@redhat.com>
 S: Maintained
 F: drivers/net/ethernet/intel/ixgbevf/
 
-- 
1.7.1